addons/interaction/functions/fnc_canTakeLauncher.sqf

```diff
@@ -0,0 +1,28 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Timi007, Cplhardcore
+ * Checks if target unit can accept given launcher.
+ * Does not check if the launcher exists in the inventory of the player.
+ *
+ * Arguments:
+ * 0: Unit that passes the launcher <OBJECT>
+ * 1: Unit to pass the launcher to <OBJECT>
+ * 2: Launcher classname <STRING>
+ *
+ * Return Value:
+ * Unit can pass launcher <BOOL>
+ *
+ * Example:
+ * [_player, _target, "launch_NLAW_F"] call ace_interaction_fnc_canTakeLauncher
+ *
+ * Public: No
+ */
+
+params ["_player", "_target"];
+if (GVAR(enableLauncherTaking) &&
+secondaryWeapon _target != "" &&
+secondaryWeapon _player == "" &&
+isNull objectParent _target &&
+isNull objectParent _player &&
+side _target == side _player) exitWith {true};
+false
```

Comment:
```suggestion

GVAR(enableLauncherTaking) &&
secondaryWeapon _target != "" &&
secondaryWeapon _player == "" &&
isNull objectParent _target &&
isNull objectParent _player &&
side _target == side _player

```

I'd personally also wrap every single statement in a lazy eval, but I haven't tested the performance impact.

Reply:
I don't think this would work when the target is unconscious, as `side` changes. Imo would be better to check if the sides have friendly relations (using the sides of the group).